deep-filtering.go

```diff
@@ -98,6 +104,43 @@ func AddDeepFilters(db *gorm.DB, objectType any, filters ...map[string]any) (*go
 
 				db = query
 
+			case string:
+				// Only wildcards if it's on and there are stars in the query
+				if !deepLike || !strings.Contains(filterType, "*") {
```

Comment:
You could consider making it more strict checking on begins with and / or ends with